primitives/traits/src/lib.rs

```diff
@@ -384,3 +385,42 @@ impl GenericStorageReader for NativeStorageReader {
         }
     }
 }
+
+pub trait RegistrarHandler<AccountId> {
```

Comment:
Comment

Reply:
Docs added for this in 9a76e31422088f5a1231e8671063cb0ed3da1581.

Reply:
_fixed_